playItAgain/css/app.css

```diff
@@ -0,0 +1,88 @@
+body{
+  background-color:orange;
+  font-family: "Monospace";
+}
+
+div{
```

Comment:
`div` is by far the most commonly used element. Putting padding on every `div` if going to hurt later. I would avoid this and just use a unique classname. `.padded-box` ?